tests/sanity/ignore-2.16.txt

```diff
@@ -0,0 +1,36 @@
+plugins/module_utils/version.py pylint!skip
+plugins/module_utils/client/discovery.py import-3.6!skip
+plugins/module_utils/client/discovery.py import-3.7!skip
+plugins/module_utils/client/discovery.py import-3.8!skip
+plugins/module_utils/client/discovery.py import-3.9!skip
+plugins/module_utils/client/discovery.py import-3.10!skip
+plugins/module_utils/client/discovery.py import-3.11!skip
```

Comment:
I believe the only thing needed to get the last sanity tests to pass is to add a line for 3.12 here, as well as for `plugins/module_utils/client/resource.py` and `plugins/module_utils/k8sdynamicclient.py`. We should also do this for the 2.17 ignore file.